extensions/blocks/contact-form/editor.scss

```diff
@@ -1,6 +1,6 @@
 
 .jetpack-contact-form {
-	padding: 10px 18px;
+	padding: 10px 0;
```

Comment:
Same same: https://github.com/Automattic/jetpack/pull/12156/files#r278600403

Reply:
Done in 8861390